content/intro-curriculum/D_Explore.md

```diff
@@ -2,16 +2,15 @@
 author: Jeffrey W. Hollister & Emily Read
 date: 2016-07-07
 slug: Explore
-draft: True
+draft: TRUE
```

Comment:
For the hugo header, it is actually "True" (not all caps)...but in this case, I think you want to lose this line?